test/ipcFixtures.hpp

```diff
@@ -328,8 +340,19 @@ TEST_P(umfIpcTest, AllocFreeAllocTest) {
     ret = umfPoolFree(pool.get(), ptr);
     EXPECT_EQ(ret, UMF_RESULT_SUCCESS);
 
+    // Test if umfCloseIPCHandle() did not corrupt the pool
+    // and if umfPoolMalloc() works correctly after umfCloseIPCHandle().
+    ptr = umfPoolMalloc(pool.get(), SIZE);
```

Comment:
But L325 already calls `umfPoolMalloc` after the `umfCloseIPCHandle` function. Why we need to check it again?

Reply:
Right. Changed to test for using an allocation after `umfCloseIPCHandle()`.